app/styles/pages/public-event.scss

```diff
@@ -185,3 +185,7 @@
 .event-map > iframe.g-map {
   height: 300px;
 }
+
+.content a {
+  text-transform: capitalize;
```

Comment:
This will capitalize all links in any content container on any page, and not just social media links on event detail page as listed in the issue

Reply:
@iamareebjamal  i will fixed this